### ISSUE TITLE
Methods for auto-updating an individual set of documentation from the source repository

### ISSUE THREAD

**Issue by nottrobin** [canonical-websites/docs.ubuntu.com#65]
Before we can implement webhooks, we need to write the methods for pulling and building a set of documentation from the source repository.

**Comment by nottrobin** [comment]
No longer needed. See https://github.com/canonical-websites/docs.ubuntu.com/issues/12#issuecomment-328897486
